include/multipass/exceptions/settings_exceptions.h

```diff
@@ -38,8 +38,8 @@ class SettingsException : public std::runtime_error
 class PersistentSettingsException : public SettingsException
 {
 public:
-    PersistentSettingsException(const QString& attempted_operation, const QString& failure)
-        : SettingsException{fmt::format("Could not {} settings: {} error", attempted_operation, failure)}
+    PersistentSettingsException(const QString& attempted_operation, const QString& detail)
+        : SettingsException{fmt::format("Unable to {} settings: {}", attempted_operation, detail)}
```

Comment:
I won't block on this, but we'll have to move away from constructing the error message like this later.

Reply:
I am not sure what the issue is. Note that `attempted_operation` could be an enum here, representing read and write ops, that we would then turn into string. Then `detail` is a separate phrase. How could I make this friendlier to translation? Perhaps an example of what problems this could cause would help me understand better.

Reply:
Basically what I mean is that "Unable to format settings" may translate to something completely different than "Unable to access settings" depending on language.

Different languages work in very different ways, building a sentence from a set of strings is bound to break at one point or another. It's a discussion to have over beers, though :)

I'm just sensitive to this because I've translated a lot of software over the years and Polish is one of the more complex languages out there so it easily highlights problems in approaches like that.

Reply:
I think I understand the idea. To avoid duplication, I think I would then try a more "square" way, e.g. "Unable to operate on settings. Operation: {}; Details: {}". But this might not feel so good for the user... Isn't this the sort of thing that one would do when preparing the code for translation? Preemptively duplicating messages in the code does not feel right to me either.

BTW, the operation here is just "read" or "read/write", "format/access" goes in the second param as a free string.